src/App.js

```diff
@@ -1,3 +1,14 @@
-export default function App() {
-  return <div>App goes here</div>;
+import React from "react";
+import "./App.css";
+import PeriodicTable from "./Components/PeriodicTable.js";
+
+function App() {
```

Comment:
You can do it like this, if you want
```suggestion
export default function App() {
```